modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc

```diff
@@ -43,8 +43,8 @@ data:
   secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
-<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
-<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
+<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`.
```

Comment:
```suggestion
<2> This depends on the GitHub product your organization is using: When using GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When using the GitHub Cloud (github.com), omit this line or enter the default `https://github.com`.
```

Reply:
@vinokurig, I could only find two major products on GitHub's website: GitHub Enterprise Server and GitHub Enterprise Cloud. So I wonder if rewriting it as follows will achieve what you suggested:
```suggestion
<2> This depends on the GitHub product your organization is using: When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL. When hosting repositories on github.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`.
```